src/main/java/org/ppijerman/ppijidentitybackend/server/dto/Branch.java

```diff
@@ -0,0 +1,41 @@
+package org.ppijerman.ppijidentitybackend.server.dto;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Calendar;
+import java.util.UUID;
+
+@Entity
+@Data
+@Table(name="BRANCH", schema="CENSUS")
+public class Branch {
+    @Id
+    @Column(name="branch_id", columnDefinition = "uuid")
+    private UUID branchID;
+
+    @Column(name="branch_name", columnDefinition="VARCHAR(50)", length = 50, nullable=false)
+    private String branchName;
+
+    @Column(name="email", columnDefinition="VARCHAR(50)", length = 50, nullable=false)
+    private String email;
+
+    @Column(name="leader", columnDefinition="VARCHAR(50)", length = 50, nullable=false)
```

Comment:
I just though again, this shall be foreign key with name `leader_id` and the datataype shall be `UUID`. This shall be foreign key with One-To-One with the Person